src/Compilers/Core/Portable/Syntax/LineDirectiveMap.LineMappingEntry.cs

```diff
@@ -66,7 +66,7 @@ public enum PositionState : byte
             // 0-based mapped span from enhanced #line directive.
             public readonly LinePositionSpan MappedSpan;
 
-            // optional 0-based character offset from enhanced #line directive.
+            // optional character offset from enhanced #line directive.
```

Comment:
Consider keeping a mention of the base ("1-based ...")

Reply:
```suggestion
            // optional 1-based character offset from enhanced #line directive.
```

Reply:
It doesn't really have a base, I think - it's just a number of characters - 0 means 0 characters, 1 means 1 character, ...